packages/flutter/test/material/flexible_space_bar_test.dart

```diff
@@ -117,6 +119,67 @@ void main() {
     expect(clipRect.size.height, minExtent);
   });
 
+  // This is a regression test for https://github.com/flutter/flutter/issues/14227
+  testWidgets('FlexibleSpaceBar sets width constraints for the title', (WidgetTester tester) async {
+    const double width = 300;
+    const double height = 100;
+    const double minExtent = 100;
+    const double maxExtent = 200;
+
+    final FlexibleSpaceBarSettings customSettings = FlexibleSpaceBar.createSettings(
```

Comment:
This test doesn't need to be so complicated, it should really be a minimal version of the test case provided with https://github.com/flutter/flutter/issues/14227.

Reply:
I simplified the test now.